test/parsers-test.js

```diff
@@ -0,0 +1,48 @@
+import chai from 'chai';
+import { expect } from 'chai';
+import parseJsonData from '../src/parsers/jsonParser.js'
+import chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised)
+
+describe('parseJsonData', function() {
+
+  it('should be a function', async function() {
+    await expect(parseJsonData).to.be.a('function')
+  })
+
+  it('should return an error for an incorrect file path', async function() {
+    await expect(parseJsonData('./badFilePath.json')).to.be.rejectedWith(Error)
+  })
+
+  it('should return an error for an incorrect file extension', async function() {
+    await expect(parseJsonData('./testData.py')).to.be.rejectedWith(Error)
+  })
+
+  it('should return an error unexpected data formats', async function() {
+    await expect(parseJsonData('./badTestData.json')).to.be.rejectedWith(Error)
+  })
```

Comment:
These are likely returning errors due to bad file paths and not the reason in the test.